secret_store/src/key_server_cluster/cluster.rs

```diff
@@ -1432,6 +1432,7 @@ pub mod tests {
 	}
 
 	#[test]
+	#[ignore]
```

Comment:
I like to see a comment explaining why a test ignored, this helps later on checking on this case and also understanding the reasoning behind this.
I think the link to #9635 would be good here.